client/src/utils/jobsUtil.js

```diff
@@ -61,67 +62,59 @@ export function getJobDisplayInformation(job, app) {
     // Improve any values with app information
     try {
       try {
-        display.systemName = getSystemName(app.exec_sys.login.host);
+        display.systemName = getSystemName(app.exec_sys.host);
       } catch (ignore) {
         // ignore if there is problem improving the system name
       }
 
-      display.applicationName = app.definition.label;
+      display.applicationName = app.definition.notes.label;
```

Comment:
Ah shoot, one more caveat here. We should only apply this if `label` is in `app.definition.notes`. There could be the case of loading an app not defined by our team, where `label` is `null`